modules/backend/formwidgets/Repeater.php

```diff
@@ -235,20 +233,13 @@ protected function processItems()
                 }
             }
         }
-
-        if (is_array($currentValue) && count($currentValue)) {
-            foreach ($currentValue as $value) {
-                $groupMap[] = array_get($value, '_group');
-            }
-        }
-
-        if (!count($groupMap)) {
-            return;
-        }
-
-        foreach ($groupMap as $index => $groupCode) {
-            $this->makeItemFormWidget($index, $groupCode);
-        }
+        
+        if (!is_array($currentValue)) return;
```

Comment:
@datune Would you mind adding braces around the conditional block, as per [PSR-2](https://www.php-fig.org/psr/psr-2/) guidelines?
```suggestion
        if (!is_array($currentValue)) {
            return;
        }
```